tradewave/templates/tradewave/marketplace-home.html

```diff
@@ -1,67 +1,13 @@
-<!DOCTYPE HTML>
-<!--
-  Escape Velocity 2.5 by HTML5 UP
-  html5up.net | @n33co
-  Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
--->
-{% load staticfiles %}
-<html>
-  <head>
-    <title>Trade Wave</title>
-    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
-    <meta name="description" content="" />
-    <meta name="keywords" content="" />
-
-    <!-- Global Variable for js -->
-    <script language="javascript">var STATIC_URL = "{{ STATIC_URL|escapejs }}";</script>
-
-    <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400italic,700,900" rel="stylesheet" />
-    <!--[if lte IE 8]><script src="js/html5shiv.js"></script><link rel="stylesheet" href="css/ie8.css" /><![endif]-->
-    <script src="{% static "js/jquery.min.js" %}"></script>
-    <script src="{% static "js/jquery.dropotron.min.js" %}"></script>
-    <script src="{% static "js/config.js" %}"></script>
-    <script src="{% static "js/skel.min.js" %}"></script>
-    <script src="{% static "js/skel-panels.min.js" %}"></script>
-    <!--noscript-->
-    <link rel="stylesheet" type="text/css" href="{% static "css/skel-noscript.css" %}"/>
-    <link rel="stylesheet" type="text/css" href="{% static "css/style.css" %}"/>
-    <link rel="stylesheet" type="text/css" href="{% static "css/style-desktop.css" %}"/>
-    <!--/noscript-->
-  </head>
-  <body class="homepage">
-
-		{% include "tradewave/header.html" %}
-
-    <!-- Intro Wrapper -->
-    <div id="intro-wrapper" class="wrapper wrapper-style1">
-      <div class="title"> Marketplace {{ name }}</div>
-      <div class="container">
-        <div class="row 150%">
-
-          {% include "tradewave/sidebar.html" %}
-
-          <div class="10u 12u(mobile) important(mobile)">
-            <div id="content">
-              <div class="row">
-                <div class="12u">
-
-                  <section id="intro">
-                    <ul class="actions">
-                      <li><a href="{% url 'tradewave:marketplace-issue' %}" class="button button-style1 button-big">Issue Credits</a></li>
-                      <li><a href="{% url 'tradewave:marketplace-redeem' %}" class="button button-style1 button-big">Redeem Credits</a></li>
-                      <li><a href="{% url 'tradewave:dashboard' %}" class="button button-style1 button-big">View Dashboard</a></li>
-                    </ul>
-                  </section>
-
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-
-    {% include "tradewave/footer.html" %}
-
-  </body>
-</html>
+{% extends "tradewave/base.html" %}
```

Comment:
@Urielar @DMalone87 This is the beauty of django template inheritance. All of the above becomes just this block of code below: